website/docs/docs/supported-data-platforms.md

```diff
@@ -3,81 +3,102 @@ title: "Supported Data Platforms"
-| Dremio                 | [Profile Setup](dremio-profile)                                              | Dremio 4.7+               | `pip install dbt-dremio`     |
-| Athena                 | [Profile Setup](athena-profile)                                              | Athena engine version 2   | `pip install dbt-athena-adapter` |
-| Vertica                | [Profile Setup](vertica-profile)                                             | Vertica 10.0+             | `pip install dbt-vertica`    |
-| AWS Glue               | [Profile Setup](glue-profile), [Configuration](glue-configs)                 | Glue 2.0+                 | `pip install dbt-glue`       |
-| Greenplum              | [Profile Setup](greenplum-profile), [Configuration](greenplum-configs)       | Greenplum 6.0+            | `pip install dbt-greenplum`  |
-| DuckDB                 | [Profile Setup](duckdb-profile)                                              | DuckDB 0.3.2              | `pip install dbt-duckdb`     |
-| SQLite                 | [Profile Setup](sqlite-profile)                                              | SQlite Version 3.0+       | `pip install dbt-sqlite`     |
-| MySQL                  | [Profile Setup](mysql-profile)                                               | MySQL 5.7 and 8.0         | `pip install dbt-mysql`      |
-| IBM DB2                | [Profile Setup](ibmdb2-profile)                                              | IBM DB2 V9fp2+            | `pip install dbt-ibmdb2`     |
+dbt connects to and runs SQL against your database, warehouse, lake or query engine. To keep things simple, we group all of these SQL-speaking things into one bucket called "data platforms". dbt can be extended to any data platform by means of a dedicated _adapter plugin_. These plugins are built as separate Python modules that dbt Core will discover them if they are installed on your system. All the adapters listed below are open source and free to use, just like dbt Core.
 
-Community-supported plugins are works in progress, and anyone is welcome to contribute by testing and writing code. If you're interested in contributing:
-- Join both the dedicated #adapter-ecosystem channel in [dbt Slack](https://community.getdbt.com/) and the channel for your adapter's data store (e.g. #db-sqlserver, #db-athena) 
-- Check out the open issues in the plugin's source repository
+To learn more about adapters, check out ["What Are Adapters"](contributing/adapter-development/1-what-are-adapters).
+
+## Adapter Installation
+
+With two exceptions [^1], all adapters listed below can be installed from PyPI using `pip install <ADAPTER-NAME>`. The installation will include `dbt-core` and any other required dependencies, which may include other adapter plugins. Read more about [installing dbt](dbt-cli/install/overview).
+
+## Adapter Taxonomy
+
+### Verified by dbt Labs
+
+We have two tiers for dbt-core adapters: "Verified" and "Community".
```

Comment:
> We have two tiers for dbt-core adapters: "Verified" and "Community".

For simplicity, I'd get rid of the table here and just explain along the lines that a subset of the adapters go through a rigorous(?) verification process across development, documentation, user experience and maintenance. These adapters earn a "Verified" designation so that users can have a certain level of trust and expectation when they use them.

Reply:
https://github.com/dbt-labs/docs.getdbt.com/pull/2032/files#r978182954